dvc/repo/experiments/__init__.py

```diff
@@ -375,11 +375,9 @@ def _commit(
         if not self.scm.is_dirty(untracked_files=True):
             raise UnchangedExperimentError(self.scm.get_rev())
 
-        rev = self.scm.get_rev()
-        if not baseline_rev:
-            baseline_rev = rev
-        checkpoint = "-checkpoint" if checkpoint else ""
-        exp_name = f"{baseline_rev[:7]}-{exp_hash}{checkpoint}"
+        rev = baseline_rev or self.scm.get_rev()
+        suffix = "-checkpoint" if checkpoint else ""
```

Comment:
`checkpoint` is already defined as being `bool`. So, changed the variable name here.